src/Routes.js

```diff
@@ -33,6 +35,7 @@ export default (
       <Route exact path="/" component={Home} />
         <Route path="/register" component={Register} />
         <Route path="/login" component={Login} />
+        <Route path="/my_space" component={MySpace} />
```

Comment:
I wasn't sure about this. But maybe a hyphen(-) is preferred over underscore(_). This has some details - [Hypher over Underscore](https://www.ecreativeim.com/blog/index.php/2011/03/30/seo-basics-hyphen-or-underscore-for-seo-urls/#:~:text=The%20short%20answer%20is%20that,treat%20an%20underscore%20that%20way.&text=This%20has%20been%20confirmed%20directly,a%20(minor)%20ranking%20benefit.)

Reply:
done. chawed it to hyphen